assets/js/theme/common/state-country.js

```diff
@@ -129,7 +130,10 @@ export default function (stateElement, context = {}, options, callback) {
 
         utils.api.country.getByName(countryName, (err, response) => {
             if (err) {
-                alert(context.state_error);
+                swal({
```

Comment:
Do you want to only execute the callback after dismissing the alert? I think `sweetalert2` is non-blocking, unlike `alert`, which stops the execution the script until it is dismissed.